cms/templates/js/settings-modal-tabs.underscore

```diff
@@ -0,0 +1,10 @@
+<ul class="settings-tabs-header">
+    <% _.each(tabs, function(tab) { %>
+    <li class="settings-tab-buttons">
+        <button class="settings-tab-button" data-tab="<%- tab.name %>" href="#"><%- gettext(tab.displayName) %></button>
```

Comment:
The gettext needs to be done where the displayName is stored so that the string is scraped.